astropy/timeseries/periodograms/lombscargle/core.py

```diff
@@ -502,14 +502,16 @@ def design_matrix(self, frequency, t=None):
         ----------
         frequency : float
             the frequency for the model
-        t : array_like or `~astropy.units.Quantity` or `~astropy.time.Time`, length n_samples
-            times at which to compute the model (optional). If not specified,
-            then the times and uncertainties of the input data are used
+        t : array_like or `~astropy.units.Quantity` or `~astropy.time.Time`
+            length ``n_samples`` times at which to compute the model
```

Comment:
As before, I think should be 'Times at which... (should have length... )' and maybe remove the (optional) and put after the types above?

Reply:
✅